### PR TITLE
AT-9 Modified MathOperations.java, add MathOperationsTest.java, created tests.

### PR DESCRIPTION
**Pull request by YehorKorotun**

### CODE REVIEW

File: src/test/java/customException/MathOperationsTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class MathOperationsTest {
+
+    @Test
+    public void sum() {
+        MathOperations mathOperations = new MathOperations();
+        Assert.assertEquals(13, mathOperations.sum(5, 8));
```
Comment (AliceIgorevna) [inline src/test/java/customException/MathOperationsTest.java:15]: why such input values were selected? base on which test design technique?

File: src/test/java/customException/MathOperationsTest.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    }
+
+    @Test
+    public void subtraction() {
+        MathOperations mathOperations = new MathOperations();
+        Assert.assertEquals(5, mathOperations.subtraction(10, 5));
```
Comment (AliceIgorevna) [inline src/test/java/customException/MathOperationsTest.java:22]: same

File: src/test/java/customException/MathOperationsTest.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    @Test
+    public void multiplication() {
+        MathOperations mathOperations = new MathOperations();
+        Assert.assertEquals(20, mathOperations.multiplication(10, 2));
```
Comment (AliceIgorevna) [inline src/test/java/customException/MathOperationsTest.java:28]: same

File: src/test/java/customException/MathOperationsTest.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    @Test
+    public void division()  {
+        MathOperations mathOperations = new MathOperations();
+        try {
+            Assert.assertEquals(2, mathOperations.division(20, 10));
+        } catch (ArithmeticException e){
+            System.out.println("You cannot divide by zero " + e);
+        }
```
Comment (AliceIgorevna) [inline src/test/java/customException/MathOperationsTest.java:38]: what this test is verifying?

**Review by AliceIgorevna — CHANGES_REQUESTED** [review]

File: src/test/java/customException/MathOperationsTest.java (hunk 1 above)
Comment (YehorKorotun) [reply src/test/java/customException/MathOperationsTest.java]: modified

File: src/test/java/customException/MathOperationsTest.java (hunk 4 above)
Comment (YehorKorotun) [reply src/test/java/customException/MathOperationsTest.java]: modified

**Pull request merged**